sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -43,6 +46,8 @@ export const PublicStory = ({ story, settings }: PublicStoryProps) => {
   const seoDescription = story.metaDescription;
   const seoImage = story.metaImage || story.coverImage;
 
+  const showCoverImage = story.coverImage && !story.hideCoverImage;
```

Comment:
we are deprecating the usage of hideCoverImage, can you please revert to how it was before?